tutorials/2d/introduction_to_2d.rst

```diff
@@ -0,0 +1,316 @@
+perform secondary actions.
+For example, keeping :kbd:`Shift` pressed while dragging a node in the Select or Move modes will 
+try to snap the node in a single axis while moving.
+
+Right clicking in the viewport provides two options to create a node or instantiate a scene 
+at the chosen position.
+If at least one node is selected, right clicking also provides the option to move the selected 
+node(s) to this position.
+
+
+Viewport has a **View** menu which provides several options to change the look of the viewport:
+
+- **Grid**: Allows you to show grids all the time, only when using snapping, or not at all. You 
+  can also toggle them with the provided option.
+- **Show Helpers**: Toggles the temporary display of an outline of the node, with the previous 
+  transform properties (position, scaling, or rotation) if a transform operation has been 
+  initiated. For `Control` nodes, it also shows the sizing parameters. Useful to see the deltas.
+- **Show Rulers**: Toggles the visibility of horizontal and vertical rulers. See 
+  :ref:`doc_introduction_to_2d_the_viewport` more on rulers.
+- **Show Guides**: Toggles the visibility of created guides. See 
+  :ref:`doc_introduction_to_2d_the_viewport` for on how to create them.
+- **Show Origin**: Toggles the display of the green and red origin lines drawn at ``x: 0, y: 0``.
+- **Show Viewport**: Toggles the visibility of the game's default 
+  viewport, indicated by an indigo-colored rectangle. It is also the default window size on desktop platforms, which can be changed by going 
+  to `Project Settings > Display > Window > Size` and setting `Viewport Width` and `Viewport Height`.
+- **Gizmos**: Toggles the visibility of `Position` (shown with cross icon), `Lock` 
+  (shown with padlock), `Groups` (shown with two squares), and `Transformation` (shown with 
+  green and red lines) indicators.
+- **Center Selection**: The same as the **Center View** button inside the viewport. Centers the selected 
+  node(s) in the view. :kbd:`F` is the default shortcut.
+- **Frame to Selection**: Similar to `Center Selection`, but also changes the zoom factor to fit the 
+  contents in the screen. :kbd:`Shift + F` is the default shortcut.
+- **Clear Guides**: Deletes all guides from the screen. You will need to recreate them if 
+  you plan to use them later. 
+- **Preview Canvas Scale**: Toggles the preview for scaling of canvas in the editor when the zoom 
+  factor or view of the viewport changes. Useful to see how the controls will look like after scaling 
+  and moving, without running the game.
+- **Preview Theme**: Allows to choose from the available themes to change the look of control items 
+  in the editor, without requiring to run the game.
+
+
+Node2D and Control node
+~~~~~~~~~~~~~~~~~~~~~~~
+
+:ref:`Node2D <class_Node2D>` is the base node for 2D. :ref:`Control <class_Control>` is the base node 
+for everything GUI. For 3D, Godot uses the :ref:`Node3D <class_Node3D>` node.
```

Comment:
```suggestion
:ref:`CanvasItem` is the base node for 2D. :ref:`Node2D <class_Node2D>` is the base node
for 2D game objects, and :ref:`Control <class_Control>` is the base node 
for everything GUI. For 3D, Godot uses the :ref:`Node3D <class_Node3D>` node.
```
As-is, this seems misleading. `CanvasItem` is the base 2D node, from the [CanvasItem docs:](https://docs.godotengine.org/en/stable/classes/class_canvasitem.html#description)
> Abstract base class for everything in 2D space. Canvas items are laid out in a tree; children inherit and extend their parent's transform. CanvasItem is extended by [Control](https://docs.godotengine.org/en/stable/classes/class_control.html#class-control) for GUI-related nodes, and by [Node2D](https://docs.godotengine.org/en/stable/classes/class_node2d.html#class-node2d) for 2D game objects.

But I'm not sure if my suggestion is the best way to clarify, I'm not a 2D expert.